rbi/core/hash.rbi

```diff
@@ -303,10 +303,6 @@ class Hash < Object
   end
   def assoc(arg0); end
 
-  # See also
-  # [`Enumerable#any?`](https://docs.ruby-lang.org/en/2.6.0/Enumerable.html#method-i-any-3F)
-  def any?(*_); end
```

Comment:
There is in fact an override of `Enumerable#any?` in `Hash`:

https://ruby-doc.org/core-2.6.3/Hash.html#method-i-any-3F

https://github.com/ruby/ruby/blob/5445e0435260b449decf2ac16f9d09bae3cafe72/hash.c#L6422

Can you instead fix / add a signature for the override as it appears in `hash.rbi`?

Omitting this definition will cause things like the hidden-definitions RBI generation to find this override as hidden, and since that autogenerated RBI definition will not have a type, it will thwart this attempt to type it.